matrix/Euler.hpp

```diff
@@ -19,34 +19,113 @@ class Dcm;
 template <typename Type>
 class Quaternion;
 
+/**
+ * Euler angles class
+ *
+ * This class describes the transformation from the body fixed frame
+ * to the inertial frame via 3-2-1 tait brian euler angles.
```

Comment:
Tait-Bran sequences can be intrinsic or extrinsic. An unambiguous description would be:

This class describes the rotation from the frame 1 to frame 2 via a 3-2-1 intrinsic Tait-Bryan rotation sequence